packages/cli/src/commands/storybook.js

```diff
@@ -31,9 +31,21 @@ export const builder = (yargs) => {
       type: 'string',
       default: 'public/storybook',
     })
+    .option('no-manager-cache', {
+      describe:
+        'Do not cache the manager UI. Useful when you have made changes to `storybook.manager.js`.',
+      type: 'boolean',
+      default: false,
+    })
 }
 
-export const handler = ({ open, port, build, buildDirectory }) => {
+export const handler = ({
+  open,
+  port,
+  build,
+  buildDirectory,
+  noManagerCache,
```

Comment:
- continued from first suggestion
```suggestion
  managerCache,
```